.github/workflows/msvc-env.yml

```diff
@@ -0,0 +1,23 @@
+name: MSVC Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: windows-latest
+    env:
+      PYTHONIOENCODING: "utf-8"
```

Comment:
fwiw, this should not be needed with meson. We already read/write all files as utf-8 and also convert to the correct locale when printing to the console.

Reply:
It's apparently needed, because I got a failure when running a test that output some Unicode glyph.

Reply:
Strange! Do you have a log? I can try to reproduce it. It should be fixed.

Reply:
```
2020-08-13T13:58:11.8100449Z # can transform 3D points
2020-08-13T13:58:11.8100656Z 
2020-08-13T13:58:11.8101069Z ok 68 - mat(translation) � point(zero) = point(translation)
2020-08-13T13:58:11.8101301Z 
2020-08-13T13:58:11.8101602Z ok 69 - mat(translation) � point3d(zero) = point3d(translation)
2020-08-13T13:58:11.8101813Z 
2020-08-13T13:58:11.8102102Z ok 70 - mat(translation) � vec3(zero) = vec3(zero)
2020-08-13T13:58:11.8102329Z 
2020-08-13T13:58:11.8102592Z 1..70
2020-08-13T13:58:11.8102795Z 
2020-08-13T13:58:11.8103193Z Traceback (most recent call last):
2020-08-13T13:58:11.8104374Z   File "c:\hostedtoolcache\windows\python\3.8.5\x64\lib\site-packages\mesonbuild\mesonmain.py", line 129, in run
2020-08-13T13:58:11.8104787Z     return options.run_func(options)
2020-08-13T13:58:11.8105155Z   File "c:\hostedtoolcache\windows\python\3.8.5\x64\lib\site-packages\mesonbuild\mtest.py", line 1045, in run
2020-08-13T13:58:11.8105470Z     return th.doit()
2020-08-13T13:58:11.8105921Z   File "c:\hostedtoolcache\windows\python\3.8.5\x64\lib\site-packages\mesonbuild\mtest.py", line 802, in doit
2020-08-13T13:58:11.8106402Z     self.run_tests(tests)
2020-08-13T13:58:11.8106787Z   File "c:\hostedtoolcache\windows\python\3.8.5\x64\lib\site-packages\mesonbuild\mtest.py", line 954, in run_tests
2020-08-13T13:58:11.8107148Z     self.drain_futures(futures)
2020-08-13T13:58:11.8107546Z   File "c:\hostedtoolcache\windows\python\3.8.5\x64\lib\site-packages\mesonbuild\mtest.py", line 969, in drain_futures
2020-08-13T13:58:11.8107887Z     result.result()
2020-08-13T13:58:11.8108258Z   File "c:\hostedtoolcache\windows\python\3.8.5\x64\lib\concurrent\futures\_base.py", line 439, in result
2020-08-13T13:58:11.8108706Z     return self.__get_result()
2020-08-13T13:58:11.8109089Z   File "c:\hostedtoolcache\windows\python\3.8.5\x64\lib\concurrent\futures\_base.py", line 388, in __get_result
2020-08-13T13:58:11.8109447Z     raise self._exception
2020-08-13T13:58:11.8109854Z   File "c:\hostedtoolcache\windows\python\3.8.5\x64\lib\concurrent\futures\thread.py", line 57, in run
2020-08-13T13:58:11.8110200Z     result = self.fn(*self.args, **self.kwargs)
2020-08-13T13:58:11.8110578Z   File "c:\hostedtoolcache\windows\python\3.8.5\x64\lib\site-packages\mesonbuild\mtest.py", line 494, in run
2020-08-13T13:58:11.8110941Z     return self._run_cmd(wrap + cmd + self.test.cmd_args + self.options.test_args)
2020-08-13T13:58:11.8111342Z   File "c:\hostedtoolcache\windows\python\3.8.5\x64\lib\site-packages\mesonbuild\mtest.py", line 625, in _run_cmd
2020-08-13T13:58:11.8111685Z     print(stdo, end='')
2020-08-13T13:58:11.8112058Z   File "c:\hostedtoolcache\windows\python\3.8.5\x64\lib\encodings\cp1252.py", line 19, in encode
2020-08-13T13:58:11.8112415Z     return codecs.charmap_encode(input,self.errors,encoding_table)[0]
2020-08-13T13:58:11.8112765Z UnicodeEncodeError: 'charmap' codec can't encode character '\u2192' in position 353: character maps to <undefined>
```

Reply:
I actually could not reproduce this on my windows machine, so it might be some CI-specific thing. I implemented this: https://github.com/mesonbuild/meson/pull/7589 going to open some test PRs to test it, please ignore :)